driver-core/pom.xml

```diff
@@ -193,8 +193,17 @@
                 <transformer implementation="org.apache.maven.plugins.shade.resource.DontIncludeResourceTransformer">
                   <resources>
                     <resource>META-INF/MANIFEST.MF</resource>
-                    <resource>META-INF/maven/io.netty/netty/pom.properties</resource>
-                    <resource>META-INF/maven/io.netty/netty/pom.xml</resource>
+                    <resource>META-INF/io.netty.versions.properties</resource>
```

Comment:
Unfortunately this is necessary, if I just provide 'pom.properties' and 'pom.xml' it successfully filters out all the netty files, but it also gets rid of the driver's pom.xml and pom.properties.   DontIncludeResourceTransformer doesn't support wildcarding, it simply checks if the resource checked against ends with any of the resources defined here.